### DISCUSSION TITLE
Your account is unable to sign up for Copilot. Please contact Support

### DISCUSSION THREAD

**Discussion by FangAk** [community/community#146338] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

![Uploading 微信图片_20241205203022.png…]()

**Comment by FangAk** [comment]
I'm using copilot as a hint Your account is unable to sign up for Copilot. Please contact Support
https://private-user-images.githubusercontent.com/139227469/350674280-e0285e05-ce03-4528-96e7-9d0eabf7b779.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzM0MDIyMTcsIm5iZiI6MTczMzQwMTkxNywicGF0aCI6Ii8xMzkyMjc0NjkvMzUwNjc0MjgwLWUwMjg1ZTA1LWNlMDMtNDUyOC05NmU3LTlkMGVhYmY3Yjc3OS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMjA1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTIwNVQxMjMxNTdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1lYjk4ODkxODMxOTEzZmQ0YWEyZDUzMjBlMWY5Njk1ZTI1YmVkZmI5YWM2MTk5MTZmYWRlMDhkYWI2NGU4ODY0JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.Ek0OpXA8ToCfYuUr2pwALLItt4UWbAGosIS_idR5SfI

**Comment by davevad93** [comment, marked as answer]
Hi @FangAk ,

This is an account related problem, in which case it's recommended to contact support by opening a ticket [here](https://support.github.com/contact/account), select “`Other`” and explain your problem with `Copilot` also attaching screenshots.

**Reply by FangAk** [reply]
I have tried to submit the work order and hope it can be solved. Thank you!

**Comment by ZhangMH2000** [comment]
你好，请问你的问题解决了吗？我也遇到了相同的问题

**Reply by LTY567** [reply]
一模一样的问题，我的账号是在tb付费加入到了一个组织，能使用高级AI，但是后面到期就无法使用了，基础的免费也无法使用了

**Reply by way0327** [reply]
问题解决了吗？我也有类似问题呢

**Reply by shengyiHuang** [reply]
> 您好，请问您的问题解决了吗？我也遇到了相同的问题

有解决吗? 我也遇到了, 在tb购买加入一个组织, 后面那个跑路了, 现在什么也用不了
